jupyter/jupyter.sh

```diff
@@ -41,12 +41,18 @@ if [ -n "${JUPYTER_CONDA_PACKAGES}" ]; then
   conda install ${JUPYTER_CONDA_PACKAGES//:/ }
 fi
 
-if [[ "${ROLE}" == 'Master' ]]; then
```

Comment:
You could still do this on master only. Up to you.

Reply:
I still like the idea of a consistent environment, so leaving it on all nodes